### PR TITLE
Add another way of string translation in js

### PR DESCRIPTION
**Pull request by anandhan-ziffity**
https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/translations/translate_theory.html#add_strings_js

## Links to Magento source code

-  ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/frontend-dev-guide/translations/translate_theory.md [hunk 1]
```diff
@@ -114,12 +114,16 @@ To ensure that the text you add in a `.js` file is collected by the i18n tool an
 
    ```javascript
    $.mage.__('<string>');
+   (or)
+   $t('<string>');
```
Comment (rogyar) [inline src/guides/v2.3/frontend-dev-guide/translations/translate_theory.md:118]: Hi @anandhan-ziffity. There's a minor nuance in this statement. In the previous list item we are linking the translate library in the following way:

```javascript
   define (['jquery', 'mage/translate'], function ($) {...});
   ```

In order to use `$t` alias, we should change the import part to the following one 

```javascript
   define (['jquery', 'mage/translate'], function ($, $t) {...});
```

**Comment by dobooth** [comment]
Hi @anandhan-ziffity Please fill out the PR description completely. Thanks!

**Comment by dobooth** [comment]
Closing as abandoned.

**Pull request closed without merging**